.github/workflows/markdownlint.yml

```diff
@@ -1,6 +1,10 @@
 name: markdownlint
 
 on:
+  push:
+    branches: [ main, metrics ]
+    paths-ignore:
+    - '**.md'
```

Comment:
Why is `*.md` being ignored here? Isn't this the exact type of file we want to trigger a `markdownlint` job when they change?

Or is there a reason it only runs on `pull_request` and not on `push`?

Reply:
This looks like a copypasta, I think it should be `paths` instead of `paths-ignore`.

Reply:
@mic-max Do you want to submit a PR fixing the typo?